kevintdd/src/main/java/ReceiptView.java

```diff
@@ -0,0 +1,37 @@
+import java.math.BigDecimal;
```

Comment:
Look at proposed changes in attachment, tests could have to be adjusted to account for corrected totals
[ReceiptView.txt](https://github.com/KTBG7/tdd/files/4956440/ReceiptView.txt)